ozaria/site/components/cinematic/PageCinematic/index.vue

```diff
@@ -15,16 +15,31 @@ module.exports = Vue.extend({
   components: {
     'cinematic-canvas': CinematicCanvas
   },
+  watch : {
+    cinematicIdOrSlug: async function() {
+      try {
```

Comment:
This code block and the code block starting on line 37 look to be the same.  These could both be consolidated into the `getCinematicData` method.